grid/social-media/followers/profile.supr

```diff
@@ -51,6 +51,32 @@ usecase GetFollowers safe {
       """
       imageUrl string
     } ]
+    """
+    Rate limit
+    Rate limit details.
+    """
+    rateLimit {
+      """
+      Total requests available
```

Comment:
The confusing think in some profile use-cases might be, that one-sdk perform will actually send more than 1 request to the Twitter API endpoints.

Reply:
Yeah, so we could either try to be smart about this and somehow try to guess the number of remaining requests for this use case, or just mention this caveat in the documentation: 1 use-case perform doesn't equal 1 request.

Reply:
> we could either try to be smart about this and somehow try to guess the number of remaining requests for this use case
I would go with this option. In case of Twitter we have enough information to calculate it.

> or just mention this caveat in the documentation: 1 use-case perform doesn't equal 1 request.
In this case we could also add number of API requests the map calls.

Reply:
We will not get into this edge case with Twitter. We will solve it once necessary by updating major version or splitting the use-case.